src/shared/components/text/Text.css.ts

```diff
@@ -0,0 +1,51 @@
+import { recipe } from '@vanilla-extract/recipes';
+import { vars } from '@styles/theme.css';
+
+export const textStyle = recipe({
+  variants: {
+    tag: {
+      head_bold_60: vars.font.head_bold_60,
+      head_bold_28: vars.font.head_bold_28,
+      head_bold_24: vars.font.head_bold_24,
+      head_sb_24: vars.font.head_sb_24,
+      head_bold_20: vars.font.head_bold_20,
+      head_sb_20: vars.font.head_sb_20,
+      head_medium_20: vars.font.head_medium_20,
+      head_regular_20: vars.font.head_regular_20,
+      body_bold_18: vars.font.body_bold_18,
+      body_medium_18: vars.font.body_medium_18,
+      body_regular_18: vars.font.body_regular_18,
+      body_bold_16: vars.font.body_bold_16,
+      body_medium_16: vars.font.body_medium_16,
+      body_regular_16: vars.font.body_regular_16,
+      body_medium_15: vars.font.body_medium_15,
+      body_regular_15: vars.font.body_regular_15,
+      body_bold_14: vars.font.body_bold_14,
+      body_medium_14: vars.font.body_medium_14,
+      body_regular_14: vars.font.body_regular_14,
+      caption_bold_13: vars.font.caption_bold_13,
+      caption_medium_13: vars.font.caption_medium_13,
+      caption_regular_13: vars.font.caption_regular_13,
+      caption_bold_12: vars.font.caption_bold_12,
+      caption_medium_12: vars.font.caption_medium_12,
+      caption_regular_12: vars.font.caption_regular_12,
+    },
```

Comment:
```suggestion
    tag: {
        ...vars.font,
    },
```
이렇게 spread 연산자 사용해도 괜찮을것 같아요!

Reply:
앗 내부에서 스프레드 사용은 안될거에요 !! 🥲🥲